app/common/controllers/saveAndContinue.js

```diff
@@ -49,8 +73,32 @@ class SaveAndContinue extends BaseController {
     res.locals.answers = questionsWithMappedAnswers.reduce(answersByQuestionCode, {})
     res.locals.rawAnswers = { ...previousAnswers, ...submittedAnswers }
 
+    // if editing a single 'record' from a multiples collection, add just that one to locals
+    if (res.locals.editMultiple && res.locals.multipleToEdit) {
+      multipleFields
+        .filter(field => field[1] === res.locals.editMultiple)
```

Comment:
For readability when accessing elements of `field` we could maybe explore something along the lines of
```
      multipleFields
        .filter(([_, someAttribute]) => someAttribute === res.locals.editMultiple)
```